test/common/secret/sds_api_test.cc

```diff
@@ -210,12 +210,17 @@ class SdsRotationApiTest : public SdsApiTestBase {
   Filesystem::MockInstance filesystem_;
 };
 
-class TlsCertificateSdsRotationApiTest : public testing::TestWithParam<bool>,
+class TlsCertificateSdsRotationApiTest : public testing::TestWithParam<std::tuple<bool, bool>>,
```

Comment:
could you put a comment on what are those booleans? It's not obvious just by looking at the code. This will help others in the future to work on the same area.